doc/source/index.rst

```diff
@@ -142,7 +142,7 @@ The Billinge Group's ``scikit-package`` has been modified from the NSLS-II scien
    :hidden:
    :caption: PROGRAMMING GUIDES
 
-   billinge-group-standards
+   programming-guides/billinge-group-standards
```

Comment:
(fix sphinx warnings/error) this was needed to import code snippets like  `.. include ../snippets/etc.rst` consistently throughout tutorials, faq, and programming guides.